UnitTests/View/ViewTests.cs

```diff
@@ -1166,15 +1166,23 @@ public void New_Initializes ()
         // BUGBUG: IsInitialized must be true to process calculation
         r.BeginInit ();
         r.EndInit ();
+    #if DEBUG
```

Comment:
I think it would be better to fix `View.ToString` to be consistent between release and debug. Both `View.Text` and `View.TItle` set `View.Id` in DEBUG. Instead, `ToString` should be smarter than this:

```cs
  public override string ToString () { return $"{GetType ().Name}({Id}){Frame}"; }
```
Instead:

```cs
    public override string ToString ()
    {
        string id = Id;

        if (string.IsNullOrEmpty (id))
        {
            if (!string.IsNullOrEmpty (Title))
            {
                id = Title;
            } else if (!string.IsNullOrEmpty (Text))
            {
                id = Text;
            }
        }
        return $"{GetType ().Name}({id}){Frame}";
    }
```

(Or similar).

Reply:
This gives me an idea.

What if there's a static counter int and, unless an explicit ID is set, it simply falls back to type name plus the next value of that integer?

Simple, clean, and at least similar to how VS would do it in the windows forms designer, with the main difference being that int is shared instead of per leaf type.

Increment is done via Interlocked.Increment for thread safety.

Reply:
And, if per-type counters are ever wanted on a type, for cosmetic purposes or whatever, it's as simple as hiding the counter field in that derived class.

Reply:
I'd support that.

Reply:
Coolio. Work for another issue, of course.

Reply:
Opened #3250 for that. Labeled as low priority. Might go ahead and handle it when doing event work that heavily touches the View class.